python/ribasim_testmodels/ribasim_testmodels/basic.py

```diff
@@ -448,13 +448,17 @@ def cyclic_time_model() -> Model:
         ],
     )
 
+    flow_boundary_geometry = Point(0.0, 0.0)
```

Comment:
```suggestion
    flow_boundary_geometry = Point(0.0, 2.0)
```

You put the FlowBoundary / area around the Basin, which I think is a bit confusing. I moved it up, such that the area is upstream of the FlowBoundary node, roughly like so:

<img width="227" height="218" alt="Image" src="https://github.com/user-attachments/assets/08180510-ab22-4271-b13e-a14dfcefecf5" />